src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequences.test.tsx

```diff
@@ -15,35 +15,56 @@
  */
 
 import React from 'react';
-import { render, getNodeText } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { configureStore, combineReducers } from '@reduxjs/toolkit';
+import { render } from '@testing-library/react';
+
+import blastFormReducer, {
+  initialState as initialBlastFormState,
+  type BlastFormState
+} from 'src/content/app/tools/blast/state/blast-form/blastFormSlice';
 
 import BlastInputSequences from './BlastInputSequences';
 
+const renderComponent = (
+  { state }: { state?: Partial<BlastFormState> } = { state: {} }
+) => {
+  const blastFormState = Object.assign({}, initialBlastFormState, state);
+  const rootReducer = combineReducers({
+    blast: combineReducers({
+      blastForm: blastFormReducer
+    })
+  });
+  const initialState = {
+    blast: { blastForm: blastFormState }
+  };
+
+  const store = configureStore({
+    reducer: rootReducer,
+    preloadedState: initialState
+  });
+
+  const renderResult = render(
+    <Provider store={store}>
+      <BlastInputSequences />
+    </Provider>
+  );
+
+  return {
+    ...renderResult,
+    store
+  };
+};
+
 describe('<BlastInputSequences />', () => {
   describe('initial state', () => {
     it('shows a single empty input', () => {
-      const { container } = render(<BlastInputSequences />);
+      const { container } = renderComponent();
       const inputBoxes = container.querySelectorAll('.inputSequenceBox');
       expect(inputBoxes.length).toBe(1);
     });
   });
 
-  describe('header', () => {
-    it('shows sequence counter, starting with 0', () => {
-      const { container } = render(<BlastInputSequences />);
-      const sequenceCounter = container.querySelector(
-        '.header .sequenceCounter'
-      );
-      expect(getNodeText(sequenceCounter as HTMLElement)).toBe('0');
-    });
-
-    it('has a control to clear all sequences', () => {
-      const { container } = render(<BlastInputSequences />);
-      const clearAll = container.querySelector('.header .clearAll');
-      expect(clearAll).toBeTruthy();
-    });
-  });
-
   describe('with added sequences', () => {
     // TODO: tests to be written after the component is connected to redux
```

Comment:
Are tests going to be added here now that the component is connected to redux?

Reply:
Added tests.